Looport/Millennium-Falcon
# Register user

# Description


The goal for this task was to by using the join form to register users in the system. After the user registered, he should be authenticated and redirected to the main page.

## What Changed


## How To Test


— Open join form via join button click or via URL `/join`
— Enter credentials and click sign up
— User should be redirected to the main page and authenticated

# Checklist
- [x] I followed all possible cases of the implemented _user flow_
